packages/utils/src/address.ts

```diff
@@ -12,12 +12,18 @@ export const privateKeyToAddress = (privateKey: string) => {
   )
 }
 
-export const publicKeyToAddress = (publicKey: string) => {
-  return '0x' + pubToAddress(Buffer.from(publicKey.slice(2), 'hex')).toString('hex')
+export const privateKeyToPublicKey = (privateKey: string) => {
+  return toChecksumAddress(
+    '0x' + privateToPublic(Buffer.from(privateKey.slice(2), 'hex')).toString('hex')
+  )
 }
 
-export const privateKeyToPublicKey = (privateKey: string) => {
-  return '0x' + privateToPublic(Buffer.from(privateKey.slice(2), 'hex')).toString('hex')
+export const publicKeyToAddress = (publicKey: string) => {
+  return toChecksumAddress(
```

Comment:
Leaving similar feedback I left for Nam on his PR that touches this:
Would be nice these methods confirmed the leading chars are '0x' before stripping them. So happens @yorhodes has added utils in this file for just this
https://github.com/celo-org/celo-monorepo/pull/1570

Reply:
`signatureUtils` has this functionality, suggest he use that.